templates/rsync.conf.lua.erb

```diff
@@ -2,6 +2,13 @@ sync {
   default.rsync,
   source = "<%= @source %>",
   target = "<%= @target %>",
+  <%- if @delete != true -%>
+    <%- if @delete == false -%>
+  delete = <%= @delete %>,
+    <%- else -%>
+  delete = "<%= @delete %>",
+    <%- end -%>
+  <%- end -%>
```

Comment:
Thanks for the PR! I really appreciate you taking the time.

The only part I can't quite follow is this block (and the similar block in `rsyncssh.conf.lua.erb`). Is there a reason we can't just add `delete = <%= @delete %>` to the templates without the surrounding conditionals?